### PR TITLE
Create mobile nav menu

### PR DESCRIPTION
**Pull request by tacettinkustu**
In this PR, our purpose is to create a curtain menu in the mobile. For this purpose, we commit this changes:
1. We created a toogle for hamburger menu
2. We added x icon to menu
3. We added event that open and close menu

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -80,30 +80,46 @@ nav {
...
+.blurred-bg {
   background: #6070ff;
   position: fixed;
+  top: 0.0000001px;
+  left: 0.0000001px;
+  width: 100%;
+  height: 100%;
+  opacity: 0.9;
+  filter: blur(5px);
+}
```
Comment (descholar-ceo) [inline style.css:92]: - [x] The blur shouldn't hide content of the site in desktop screen, I think you should consider that as well
Screenshot from 2021-06-22 10-45-19

File: style.css [hunk 2]
```diff
@@ -80,30 +80,46 @@ nav {
...
+  flex-direction: column;
+  top: 0.0000001px;
+  left: 0.0000001px;
   width: 100%;
   height: 100%;
-  top: 0.000000001px;
-  left: 0.000000001px;
-  right: 0.000000001px;
-  opacity: 0.98;
-  filter: blur(8px);
```
Comment (descholar-ceo) [inline style.css]: - [x] [optional] Practically the navbar should be attached with the top edge of the viewport, currently there is an empty space between the top edge of the navbar and the viweport, would you fix that?

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @tacettinkustu :handshake: :handshake: :handshake: , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [x] I suggest you update the live demo link presented in your readme file 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (tacettinkustu) [reply style.css:92]: I missed this problem. And fixed.

File: style.css (hunk 2 above)
Comment (tacettinkustu) [reply style.css]: When I make the top value zero, there is a linter error. But I will try that again. Thank you.

**Comment by tacettinkustu** [comment]
> Hi @tacettinkustu 🤝 🤝 🤝 ,
> 
> Good job so far!
> There are some issues that you still need to work on to go to the next project but you are almost there!
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> * [x]  I suggest you update the live demo link presented in your readme file
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Thank you for your reviews.

**Review by Dipeshtwis — APPROVED** [review]
# Status: Approved ✔️

## Hi @tacettinkustu, 

Good job on implementing the suggested changes. Well Done.
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
---------

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
_**Dipesh Kumar**_

--------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**